### PR TITLE
Add missing files

### PR DESCRIPTION
**Pull request by wbloszyk**
Revert missing file while #1968

### CODE REVIEW

**Comment by jordisala1991** [comment]
Please add a comment like the previous one: https://github.com/sonata-project/SonataMediaBundle/blob/db9aa7b6b49a3b2977c4ff15766698f899234b72/src/Resources/config/no_driver_admin.xml#L3

File: src/Resources/config/no_driver_admin.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->services();
```
Comment (jordisala1991) [inline src/Resources/config/no_driver_admin.php]: I dont think this line is needed

Comment (wbloszyk) [reply src/Resources/config/no_driver_admin.php]: In MediaBundle exists some block which require Admin services. 
IMO will be nice to allow register admin service with `manager_type: no_driver`.
https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/getting_started/creating_an_admin/#step-2-register-the-admin-class 

Then in `no_driver_admin.php` we should add this admins. 

`no_driver` should display Admin panel with alert about missing driver.

Comment (jordisala1991) [reply src/Resources/config/no_driver_admin.php]: I dont understand what you mean. 

I think there is a bug about this file missing. But the line I mention it is not needed. 

The other thing you mention is maybe a feature for 3.x. Not for 4.x.

Comment (wbloszyk) [reply src/Resources/config/no_driver_admin.php]: I mean we should integrate `no_driver` in AdminBundle too. Then we should register admin services in `no_driver_admin.php`.

Agree. Services should be add in 3.x

File: src/Resources/config/no_driver_admin.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+/*
```
Comment (jordisala1991) [inline src/Resources/config/no_driver_admin.php:13]: Did you try this change? I never saw a services file in php without the return static function.

Comment (wbloszyk) [reply src/Resources/config/no_driver_admin.php:13]: It is working on my demo.

**Review by jordisala1991 — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**